src/xo-app/dashboard/overview/index.js

```diff
@@ -303,7 +304,9 @@ export default class Overview extends Component {
                   <Icon icon='pool' /> {_('poolPanel', { pools: this.props.nPools })}
                 </CardHeader>
                 <CardBlock>
-                  <p className={styles.bigCardContent}>{this.props.nPools}</p>
+                  <p className={styles.bigCardContent}>
+                    <Link to={{pathname: '/home', query: {t: 'pool'}}}>{this.props.nPools}</Link>
```

Comment:
I did as documented [here](https://github.com/reactjs/react-router/blob/master/docs/API.md#link), you'r talking about something like this ? `<Link to='/home?t=pool'/>`